Gems/AtomLyIntegration/CommonFeatures/Code/Source/Debug/RenderDebugEditorComponent.cpp

```diff
@@ -23,7 +23,7 @@ namespace AZ::Render
             if (AZ::EditContext* editContext = serializeContext->GetEditContext())
             {
                 editContext->Class<RenderDebugEditorComponent>(
-                    "DebugRendering", "Controls for debugging rendering.")
+                    "Debug Rendering", "Controls for debugging rendering.")
```

Comment:
Changing this name may actually impact Python tests using this API call: `FindComponentTypeIdsByEntityType`. This function allows tests to retrieve the TypeId used in other APIs from the component name, so changing that value would break all tests referring to it.

We do have an alias/deprecated name system for properties AFAIK, but not for component names. We may have to add one to allow this kind of operation.

Reply:
There don't appear to be any references in our own python files to "DebugRendering"